lib/ansible/modules/network/netconf/netconf_config.py

```diff
@@ -152,191 +193,178 @@
     returned: success
     type: list
     sample: ['urn:ietf:params:netconf:base:1.1','urn:ietf:params:netconf:capability:confirmed-commit:1.0','urn:ietf:params:netconf:capability:candidate:1.0']
-
+backup_path:
+  description: The full path to the backup file
+  returned: when backup is yes
+  type: string
+  sample: /playbooks/ansible/backup/config.2016-07-16@22:28:34
+diff:
+  description: If --diff option in enabled while running, the before and after configration change are
+               returned as part of before and after key.
+  returned: when diff is enabled
+  type: string
+  sample: /playbooks/ansible/backup/config.2016-07-16@22:28:34
 '''
 
-import traceback
-import xml.dom.minidom
-
-from xml.etree.ElementTree import fromstring, tostring
-
-try:
-    import ncclient.manager
-    HAS_NCCLIENT = True
-except ImportError:
-    HAS_NCCLIENT = False
-
+from ansible.module_utils._text import to_native, to_text
```

Comment:
It doesn't look like you're using `to_native` anymore. I wouldn't normally nitpick for leftover imports, but we'd like to discourage use of to_native without a pretty good reason, and removing the import would make adding it back just a little bit more work.